docs/resources/project.md

```diff
@@ -128,3 +128,9 @@ Required:
 Optional:
 
 - **description** (String)
+
+## Note
+### Repository assignment & unassignment Limitations
+- Project provider has limitations while assigning or de-assigning large number of repositories through repos attribute. With large number of repos you might observe system disruptions and all the repositories might not get assigned/unassigned to/from desired project.
+- Try to limit number of repos with in 10 to get a desired result.
```

Comment:
Why don't we experiment and see where that limit is, then actually put a validator on it with an override?

Reply:
I mean, if we overload it and we shatter the service, then that's just bad

Reply:
Found out the yield point on Partnership engineering environment. 100 Repos and 15 retries.

Reply:
validator implemented with an override from environment.